audio/alsa-plugins/DEPENDS

```diff
@@ -1,4 +1,5 @@
 depends alsa-lib
+depends speex
```

Comment:
Speex should be an optional dependency, please take a look at "--with-speex={builtin|lib|no}" configure flag.